lib/datasource/docker/index.ts

```diff
@@ -199,7 +199,6 @@ async function getManifestResponse(
   repository: string,
   tag: string
 ): Promise<GotResponse> {
-  logger.debug(`getManifestResponse(${registry}, ${repository}, ${tag})`);
```

Comment:
Are we sure to remove those `debug` logging?

Reply:
I don't mind it. We were never consistent anyway.